app/views/countries/partials/_habitat-representation.html.erb

```diff
@@ -1,13 +1,13 @@
 <section class="container section-padding--top">
   <div class="flex">
-    <h2>
+    <h3>
```

Comment:
Not that this is a big issue, but I feel that this should be an h2 as it's representing a primary section of the page.

Reply:
Yes, this is now a subsection and the grey backgrounds do need updating. This is planned to be looked at, but is being left as is for the moment. I suspect it's one to go through Charlotte, but is being left until after launch.